client/src/components/AlumniNetworking.js

```diff
@@ -34,9 +43,39 @@ export default class AlumniNetworking extends Component {
         this.setState({display: display})
     }
 
+    async openConversation(e) {
+        let conversationTarget = e.currentTarget.getAttribute('conversationid')
+        let conversation = await this.fetchConversation(conversationTarget)
+        this.setState({
+                activeConversation: conversation.conversation,
+                conversationModalOpen: true
+            })
+    }
+
+    fetchConversation(conversationTarget) {
+        return (makeCall({id: conversationTarget, timezone: this.state.timeOffsetHours},
+            '/conversations/one/' + this.props.userDetails._id, 'patch'))
+    }
+
+    async closeConversationModal() {
+        let conversations = await this.getConversations()
+        this.setState({
+            conversations: conversations.conversations,
+            conversationModalOpen: false, 
+            activeConversation: null
+        })
+        this.createDisplay(conversations.conversations)
+    }
+
     constructConversation(conversation) {
         let userIndex = conversation.alumni.findIndex(item => item._id === this.props.userDetails._id);
         let recipientIndex = (userIndex + 1) % 2;
+        let formattedMessage = conversation.messages[0].message.split('\n')
+        formattedMessage =  (
+            <p style={{'wordWrap': 'break-word', 'hyphens': 'auto', 'width': '50vw'}}>
```

Comment:
This styling is because Semantic UI react components don't prevent words from overflowing

Reply:
Very neat!

Reply:
Can we add this comment/clarification in-line?